app/schema/get-search-api-data-schema.tsx

```diff
@@ -0,0 +1,11 @@
+import z from "zod";
```

Comment:
I'd name this file `discogs.schemas.ts` and move it next to your SearchApi/Discogs file